test/support/database.js

```diff
@@ -1,3 +1,5 @@
+const ANALYTICS_DATA_TABLE_NAME = "analytics_data"
```

Comment:
This is a total nit and doesn't block the merge, but is it possible for you to access this value from the original file being tested? `require("src/write-results-to-db.js").ANALYTICS_DATA_TABLE_NAME` or something like that? (I forget Node's rules around variable accessiblity through `require` and how it intersects with `const`.)